### PR TITLE
force running cluster up tests if cluster up code changes

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@stevekuznetsov @csrwng ptal

File: cluster/ci/config/prow/config.yaml [hunk 1]
```diff
@@ -151,6 +151,7 @@ presubmits:
   - name: test_pull_request_origin_extended_clusterup
     agent: jenkins
     always_run: false
+    run_if_changed: '^(pkg/oc/bootstrap)|^(test/extended/clusterup.sh)'
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]: `'^(pkg/oc/bootstrap|test/extended/clusterup.sh)' ?

Comment (bparees) [reply cluster/ci/config/prow/config.yaml]: does it matter? I find the parens offer a nice separator.

Comment (stevekuznetsov) [reply cluster/ci/config/prow/config.yaml]: no

**Pull request merged**